src/main/frontend/src/app/staticData.js

```diff
@@ -0,0 +1,27 @@
+// TODO: This is static and should be replaced when the Lumen's endpoints are available.
+// All of these static variables are going to be moved to the Redux store and the components will listen to changes.
+const releaseDate = '06/03/2022';
+
+
+const sectionArray = [
+  {name: 'Course Contents', items: ['About This Course.', 'Course Contents at a Glance.', 'Course Learning Outcomes.']},
+  {name: 'Course Contents', items: ['About This Course.', 'Course Contents at a Glance.', 'Course Learning Outcomes.']},
+  {name: 'Course Contents', items: ['About This Course.', 'Course Contents at a Glance.', 'Course Learning Outcomes.']},
+];
+
+export const COURSE_ARRAY = [
```

Comment:
I'll make a note about this ini the channel.

Reply:
@smith750 This is just a mock object, I'll replace the object by the schema when the endpoint is available, thanks!